index.css

```
@@ -0,0 +1,117 @@
+body {
+    min-width: 1920;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-size: 16px;
+}
+
+.wrapper {
+    width: 1120;
+    height: 700;
+    display: block;
+    margin-left: 400px;
+    margin-right: auto;
+}
+
+.hero_wrapper {
+    width: 1120;
+    height: 700;
+    display: block;
+    margin: auto 400px;
+    text-align: center;
+}
+
+.hero_wrapper p {
+    width: 500px;
+    text-align: center;
+    margin: 50px auto 100px auto;
+}
+
+header {
+    background-color: #EEEFF1;
+    margin-bottom: 50px;
+}
+
+header p {
+    font-size: 16px;
+    max-width: 540px;
+    line-height: 26px;
+    font-weight: 400;
+}
+
+menu {
+    word-spacing: 20px;
+    padding: 80px 0;
+}
+
+button {
+    background-color: #34547A;
+    color: #fff;
+    font-size: 16px;
+    text-align: center;
+    width: 225px;
+    height: 65px;
+    margin-top: 60px;
+    margin-bottom: 140px;
+}
+
+
+h1 {
+    font-size: 48px;
+    margin-top: 60px;
+    line-height: 78px;
+}
+
+h2 {
+    font-size: 32px;
+    margin: 100px auto 50px auto;
+    line-height: 32px;
+    text-align: center;
+}
+
+section {
+    background-color: #34547A;
+    display: block;
+    height: 250px;
+    color: #fff;
+}
+
+.new_wrapper {
+    width: 1120;
+    display: block;
+    margin-left: 400px;
+    margin-right: auto;
+}
```

```suggestion
.new_wrapper {
    width: 1120px;
    display: block;
    margin-left: auto;
    margin-right: auto;
}
```
і тоді буде працювати як потрібно